### PR TITLE
Created sync for referral links from saasquatch to hubspot by creating new property fields in the contacts Object

### PR DESCRIPTION
**Pull request by jasperhale342**
The feature file seems a bit long to me so I was wondering how much detail it should be

### CODE REVIEW

File: src/integration/HubspotApiModel.ts [hunk 1]
```diff
@@ -63,6 +63,74 @@ export class HubspotApiModel {
...
+    public async objectHasProperty(objectType: string, propertyName: string){
+        const readPropertyURL = 'https://api.hubapi.com/crm/v3/properties/' + objectType + '/' + propertyName; 
+        try {
+         const response = await axios.get(readPropertyURL, {
+             params: {
+                 hapikey: this.HAPIKEY
+             }
+         });
+            if (response.status==200 ){
+                console.log("=== PROPERTY EXIST IN OBJECT TYPE " + objectType + " ===")
```
Comment (tolkamps1) [inline src/integration/HubspotApiModel.ts]: Looks good! Maybe we can remove the console.logs for the success cases, and only log errors/issues?

**Review by tolkamps1 — APPROVED** [review]

**Review by EvanDungate — COMMENTED** [review]
Ran this branch and sharelinks show up in hupspot when a new participant is created. I would change the namespacing to be more descriptive though or have a space between the name and a capital "S" in Saasquatch something like:

 let newProgramShareLink = "Saasquatch Sharelink: " + key.replace(/\W/g, '') + "Program";
 
 Also, I didnt see sharelinks show up in hubspot after I created a new participant. Not sure if this was supposed to happen or if that should be a new ticket with the expected behaviour:
Given: the integration is running
When a new contact is created in hubspot
Then a new participant is created in saasquatch with sharelinks
and those sharelinks show up in matching contact's custom properties

**Comment by jasperhale342** [comment]
> Ran this branch and sharelinks show up in hupspot when a new participant is created. I would change the namespacing to be more descriptive though or have a space between the name and a capital "S" in Saasquatch something like:
> 
> let newProgramShareLink = "Saasquatch Sharelink: " + key.replace(/\W/g, '') + "Program";
> 
> Also, I didnt see sharelinks show up in hubspot after I created a new participant. Not sure if this was supposed to happen or if that should be a new ticket with the expected behaviour:
> Given: the integration is running
> When a new contact is created in hubspot
> Then a new participant is created in saasquatch with sharelinks
> and those sharelinks show up in matching contact's custom properties

Sounds good. Yeah the scope of this ticket was to just post referrals links to hubspot when a new user is created. Also, I did snake case because property names cannot have any uppercase letters

**Review by EvanDungate — APPROVED** [review]
Looks good thank you!

**Pull request merged**